rust/basic_bitcoin/src/basic_bitcoin/src/ecdsa_api.rs

```diff
@@ -0,0 +1,53 @@
+use crate::types::*;
+use ic_cdk::{call, export::Principal};
+
+const KEY_NAME: &str = "test";
```

Comment:
[As suggested @THLO](https://github.com/dfinity/examples/pull/281/files/aab31d9a4aecfcd7856113b6eb354926a14d2747#r916953714), we should update the key before we release the code. According to [this proposal](https://nns.ic0.app/#/proposal/68945), we should use `test_key_1` instead of `test`. After having tested, making this modification doesn't involve any problem when using the mock tECDSA canister locally. Do you confirm that `test_key_1` should be the `KEY_NAME` as you tested on the IC ?

Reply:
Yes, I confirm.